src/Components/Endpoints/src/RazorComponentEndpointInvoker.cs

```diff
@@ -88,30 +94,32 @@ private async Task RenderComponentCore()
         await writer.FlushAsync();
     }
 
-    private Task<bool> TryValidateRequestAsync(out bool isPost)
+    private Task<bool> TryValidateRequestAsync(out bool isPost, out string? handler)
     {
+        handler = null;
         isPost = HttpMethods.IsPost(_context.Request.Method);
         if (isPost)
         {
-            return Task.FromResult(TrySetFormHandler());
+            return Task.FromResult(TrySetFormHandler(out handler));
         }
 
         return Task.FromResult(true);
     }
 
-    private bool TrySetFormHandler()
+    private bool TrySetFormHandler([NotNullWhen(true)] out string? handler)
     {
-        var handler = "";
+        handler = "";
```

Comment:
Why is this initialized to an empty string as opposed to `null`?

Reply:
Because "" is the name for the default form handler.